### PR TITLE
Migration of account generation page to support pages

### PR DESCRIPTION
**Pull request by filippoweb3**
Left some tiny intro and a list of ways to create accounts with links to support pages

### CODE REVIEW

File: docs/learn/learn-account-generation.md [hunk 1]
```diff
@@ -8,415 +8,21 @@ slug: ../learn-account-generation
 ---
 
 An address is the public part of a Polkadot account. The private part is the key used to access this
-address. The public and private parts together make up a Polkadot account.
+address. The public and private parts together make up a Polkadot account. You can think about your public address like your mailbox and your private key like your key to open your mailbox. Everybody can send you information to your public address, but only you can access the box and read such information. In other words, everybody can know your public address but you should keep your private key secret.
```
Comment (profradha) [inline docs/learn/learn-account-generation.md]: I'm thinking if there are better words to to explain this than just saying public and private part.

Rephrased it a little bit. Feel free to edit it further

```suggestion
address. The public and private parts together make up a Polkadot account. You can think of the public address  of your account like your mailbox and the private key like the key to open that mailbox. Anybody can send mail to your mailbox, but only you can access them as only you have access to its key. In the context of Polkadot accounts,  anybody can send tokens to your public address but only you can transact with them using your private key. That is why you should keep your private key secret.
```

**Review by DrW3RK — CHANGES_REQUESTED** [review]
I have some feedback that can help with future PRs as well.

1. It is important to ensure the content being deleted on the Wiki is present in the support article being referenced. 
2. It is okay to have redundant information on the Wiki page as well as the Support page. The ultimate goal is to have a coherent reading experience for the user

You have put a lot of effort to create the password reset section on the Wiki, but deleted it without ensuring it is transmitted to the support page. I tried to retain the backbone of this Wiki document with relevant references to the support articles. Let me know what you think from the perspective of a new reader landing on this Wiki page.

**Comment by filippoweb3** [comment]
I overlooked it and did not notice I deleted all that part I worked on! My apologies, I agree the page must be like you mentioned. I am going through it and check if it need further additions / rearrangements.

File: docs/learn/learn-account-generation.md [hunk 2]
```diff
@@ -8,16 +8,17 @@ slug: ../learn-account-generation
...
 
 An address is the public part of a Polkadot account. The private part is the key used to access this
-address. The public and private parts together make up a Polkadot account.
+address. The public and private parts together make up a Polkadot account. You can think of the public address  of your account like your mailbox and the private key like the key to open that mailbox. Anybody can send mail to your mailbox, but only you can access them as only you have access to its key. In the context of Polkadot accounts,  anybody can send tokens to your public address but only you can transact with them using your private key. That is why you should keep your private key secret.
+
 
 There are several ways to generate a Polkadot account:
 
-- [Polkadot{.js} Browser Extension](#polkadotjs-browser-extension) **RECOMMENDED FOR MOST USERS**
+- [Polkadot{.js} Browser Extension](#polkadot-js-apps) **RECOMMENDED FOR MOST USERS**
```
Comment (profradha) [inline docs/learn/learn-account-generation.md]: Have you mistakenly pointed this ref link to point to Polkadot JS Apps?

```suggestion
- [Polkadot-JS Browser Extension](#polkadot-js-browser-extension) **RECOMMENDED FOR MOST USERS**
```

File: docs/learn/learn-account-generation.md [hunk 3]
```diff
@@ -39,98 +40,25 @@ security trade-offs. Please review this page carefully before making your accoun
...
-[mnemonic password of four to five words](https://xkcd.com/936/). These are nearly impossible for
-computers to guess due to the number of combinations possible, but much easier for humans to
-remember.
+## Polkadot JS Browser Extension
 
-## Polkadot{.js} Browser Extension
-
-The Polkadot{.js} Extension provides a reasonable balance of security and usability. It provides a
+The Polkadot JS Extension provides a reasonable balance of security and usability. It provides a
 separate local mechanism to generate your address and interact with Polkadot.
```
Comment (profradha) [inline docs/learn/learn-account-generation.md]: Staying consistent with Glossary https://support.polkadot.network/support/solutions/articles/65000110873-glossary and at the same time, preserving the link to this section referenced on third party websites
```suggestion
The Polkadot-JS Browser Extension (simply referred to as Polkadot Extension) provides a reasonable balance of security and usability. It provides a
separate local mechanism to generate your address and interact with Polkadot.
```

File: docs/learn/learn-account-generation.md [hunk 4]
```diff
@@ -39,98 +40,25 @@ security trade-offs. Please review this page carefully before making your accoun
...
 
-This method involves installing the Polkadot{.js} plugin and using it as a “virtual vault," separate
+This method involves installing the Polkadot JS extension and using it as a “virtual vault," separate
```
Comment (profradha) [inline docs/learn/learn-account-generation.md]:
```suggestion
This method involves installing the Polkadot Extension and using it as a “virtual vault," separate
```

File: docs/learn/learn-account-generation.md [hunk 5]
```diff
@@ -39,98 +40,25 @@ security trade-offs. Please review this page carefully before making your accoun
...
-If you would like to know more or review the code of the plugin yourself, you can visit the
-[GitHub source repository](https://github.com/polkadot-js/extension).
-
-After installing the plugin, you should see the orange and white Polkadot{.js} logo in the menu bar
-of your browser.
-
-![plugin-02](../assets/accounts/polkadot_plugin_js_02.jpg)
-
-### Create Account
+:::info For the tutorial on how to create an account using Polkadot JS browser extension, visit [this support article](https://support.polkadot.network/support/solutions/articles/65000098878-how-to-create-a-dot-account).
```
Comment (profradha) [inline docs/learn/learn-account-generation.md]:
```suggestion
:::info For the tutorial on how to create an account using Polkadot Extension, visit [this support article](https://support.polkadot.network/support/solutions/articles/65000098878-how-to-create-a-dot-account). 
```

File: docs/learn/learn-account-generation.md [hunk 6]
```diff
@@ -172,36 +100,24 @@ networks without having to set different passwords to access such accounts.
 
 ### Set Address for Polkadot Mainnet
 
-Now we will ensure that the addresses are displayed as Polkadot mainnet addresses.
+In Polkadot you can use the same address on multiple chains. To switch between chains you can follow the guidelines in [this support article](https://support.polkadot.network/support/solutions/articles/65000103707-can-i-use-the-same-account-on-polkadot-kusama-and-parachains-). The [Accounts page](learn-accounts.md#address-conversion-tools) has a tool you can use to
+convert your address between the different chain formats. 
 
-Click on "Options" at the top of the plugin window, and under "Display address format for" select
-"Polkadot Relay Chain".
+:::info For privacy reasons, we recommend creating a new address for each chain you're using.
```
Comment (profradha) [inline docs/learn/learn-account-generation.md:107]: Reasoning behind the recommendation 

```suggestion

Your address' format is only visual - the data used to derive this representation of your address
are the same, so you can use the same address on multiple chains. However, for privacy reasons, we
recommend creating a new address for each chain you're using.
```

**Review by DrW3RK — APPROVED** [review]
Looks good. Made a few suggestions. Please address them and then we can merge this

**Pull request merged**